source/backup-cursor.rst

```diff
@@ -0,0 +1,97 @@
+.. _backup-cursor:
+
+============================================================
+$backupCursor and $backupCursorExtend aggregation stages
+============================================================
+
+``$backupCursor`` and ``$backupCursorExtend`` aggregation stages expose the WiredTiger API which allows making consistent backups. Running these stages allows listing and freezing the files so you can copy them without the files being deleted or necessary parts within them being overwritten. 
+
+* ``$backupCursor`` outputs the list of files and their size to copy.  
+* ``$backupCursorExtend`` outputs the list of WiredTiger transaction log files that have been updated or newly added since the ``$backupCursor`` was first run. Saving these files enables restoring the database to any arbitrary time between the ``$backupCursor`` and ``$backupCursorExtend`` execution times.
+
+They are available in |PSMDB| starting with version 4.4.6-8. 
+
+Percona provides `Percona Backup for MongoDB (PBM) <https://www.percona.com/doc/percona-backup-mongodb/index.html>`_ – a light-weight open source solution for consistent backups and restores across sharded clusters. PBM relies on these aggregation stages for physical backups and restores. However, if you wish to develop your own backup application, this document describes the ``$backupCursor`` and ``$backupCursorExtend`` aggregation stages.
+
+Usage 
+=============
+
+You can run these stages in any type of MongoDB deployment. If you need to a back up a single node in a replica set, first run the ``$backupCursor``, then the ``$backupCursorExtend`` and save the output files to the backup storage.
+
+To make a consistent backup  of the whole replica set / a sharded cluster, run both aggregation stages on one node from each replica set. It can be either the primary or the secondary node. Note that since the secondary node may lag in syncing the data from the primary one, you will have to wait for the exact same time before running the ``$backupCursorExtend``.
```

Comment:
Please change to 
To make a consistent backup of the sharded cluster, run both aggregation stages on one node from each replica set.